debian/acrn-board-inspector/setup.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
This is the way I resolved the missing setup.py. Since setuptools/distutils support is already deprecated from Python point of view (https://peps.python.org/pep-0632/) but Debian still uses it, but is already working on a solution, it might be worth to wait until the Debian people come up with their final solution. This will need changes anyway!